app/celery/lookup_recipient_communication_permissions_task.py

```diff
@@ -58,18 +61,37 @@ def recipient_has_given_permission(
         notification_id: str,
         notification_type: str,
         communication_item_id: str
-) -> bool:
+) -> str:
+    default_send_flag = True
+    communication_item = None
     identifier = RecipientIdentifier(id_type=id_type, id_value=id_value)
 
-    communication_item = get_communication_item(communication_item_id)
+    try:
+        communication_item = get_communication_item(communication_item_id)
+    except NoResultFound:
+        current_app.logger.info('No communication item found for notification %s', notification_id)
 
     try:
+        # get default send flag when available
+        if communication_item is not None:
+            default_send_flag = communication_item.default_send_indicator
+
         is_allowed = va_profile_client.get_is_communication_allowed(
```

Comment:
If communication_item is None, does this always raise an exception?  If so, only call it if `communication_item is not None`.